api/output_policy.go

```
@@ -88,96 +86,22 @@ func (d *OutputPolicyError) buildSamplePolicy() (string, error) {
 }
 
 // Determine whether the given path requires the sudo capability
-func isSudoPath(client *Client, path string) (bool, error) {
-	sudoPaths, err := getSudoPaths(client)
-	if err != nil {
-		return false, fmt.Errorf("unable to retrieve list of paths that require sudo capability: %v", err)
-	}
-	if sudoPaths == nil || len(sudoPaths) < 1 {
-		// OpenAPI spec did not return any paths that require sudo,
-		// but the user probably still shouldn't see an error.
-		return false, nil
-	}
-
-	// Return early if the path is clearly one of the sudo paths.
+func isSudoPath(path string) (bool, error) {
```

I don't think we need to return an error